text/0006-architecture-review-process.md

```diff
@@ -57,28 +58,20 @@ For both a Roadmap Review and a Design Review the **timetable** is as follows:
       - A step back to discuss if the proposal is right in the wider context
     * A lightweight review might not need the input of others, and ultimately, might not even need a specific review meeting.
 
-The review meeting should be a discussion of the issues, but should avoid feeling pressured to make a decision. The goal is to understand the issues raised by the question(s) and the background from the review packet, and to add to it wisdom from the people at the review. The meeting may decide to alter the questions and undergo a subsequent review. We should be very open about conversations that happen after the review meeting.
+The review meeting should be a discussion of the issues, but should avoid feeling pressured to make a decision. The goal is to understand the issues raised by the question(s) and the background from the review packet, and to add to it wisdom from the people at the review.
```

Comment:
I know this is existing verbiage, but here too I worry about how potential reviewers will react to being told how to feel (i.e. that they "should avoid feeling pressured"). I would reword to emphasize just that reviewers don't need to make a decision in the meeting (without reference to what they may or may not feel), something like:

> The review meeting should be a discussion of the issues. Reviewers don't need to make a decision at the meeting. The goal is to understand the issues raised by the question(s) and the background from the review packet, and to add to it wisdom from the people at the review.

Reply:
Fixed in 3f828ffc3981de51cd31729115d3f51efed126d9